properties/size/breakpoint.json

```diff
@@ -0,0 +1,26 @@
+{
+  "size": {
+    "breakpoint": {
+      "phone": {
+        "value": "480",
```

Comment:
I don't see any use for the "phone" breakpoint. What's the intention here? I can see how tablet is devices 768 and up, desktop is devices 1024 and up, but phone being devices 480 and up doesn't make any sense. Most phones are actually smaller than 480. Is the intention to target devices smaller than 480? If so, what's the use case for that? For a mobile-first approach, I would setup my CSS to default to mobile, and then use the 768 and 1024 breakpoints to add tablet and desktop styles.

Reply:
@RyanMacBern the numbers are meant to represent `up to`. So in this case `mobile` would be `0 - 480px` and so on.

Reply:
That would make tablet 480 - 768? Typically tablet is 768 is up.